pages/challenge.tsx

```diff
@@ -0,0 +1,32 @@
+import Head from "next/head"
+
+export default function Web() {
+  return (
+    <>
+      <Head>
+        <meta property="og:url" content="https://next-enterprise.vercel.app/" />
+        <meta
+          property="og:image"
+          content="https://raw.githubusercontent.com/Blazity/next-enterprise/main/project-logo.png"
+        />
+        <meta property="og:image:width" content="1200" />
+        <meta property="og:image:height" content="630" />
+        <meta name="twitter:card" content="summary_large_image" />
+        <title>Oles eleven</title>
+      </Head>
+      <section className="bg-white dark:bg-gray-900">
+        <div className="mx-auto grid max-w-screen-xl px-4 py-8 text-center lg:py-16">
+          <div className="mx-auto place-self-center">
+            <h1 className="mb-4 max-w-2xl text-4xl font-extrabold leading-none tracking-tight dark:text-white md:text-5xl xl:text-6xl">
+              11 Miles challenge
+            </h1>
+            <p className="mb-6 max-w-2xl font-light text-gray-500 dark:text-gray-400 md:text-lg lg:mb-8 lg:text-xl">
+              This challenge is to complete 11 miles (110 kilometers) in under 24 hours by walking and running. It must
+              include atleast one hour of running. it can not include any othe form of transportation.
```

Comment:
Suggest changing text to this:

The 11 Scandinavian Miles is the challenge to traverse 110 kilometers by walking and running within 24 hours. The two rules are simple: 1) no other mode of transportation than your own legs, and 2) at least 10 kilometers of the distance must be traversed while running.

Reply:
done